src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.scss

```diff
@@ -0,0 +1,3 @@
+.form-group.ng-invalid .invalid-feedback {
```

Comment:
Maybe we should remove this rule from here and append it to `sytles.scss:400`, so future occurrences also benefit from the style.

Reply:
I only had to do this since the `has-error` class was never applied to the embedded form group. I just didn't want to potentially break something else by changing something at the global level. If you think it's ok to apply 'display: block' when the `invalid-feedback` parent class also has `ng-invalid`, I can move it up.